packages/extester/src/util/codeUtil.ts

```diff
@@ -28,6 +28,8 @@ export interface RunOptions {
     logLevel?: logging.Level;
     /** try to perform all setup without internet connection, needs all requirements pre-downloaded manually */
     offline?: boolean;
+    /** path to unbundled extension sources */
+    extensionDevelopmentPath?: string;
```

Comment:
is this needed if we directly using CWD?

Reply:
Same as above. I will try it and give updates here.

According to [vscode-docs](https://vscode-docs.readthedocs.io/en/stable/extensions/debugging-extensions/#launching-your-extension), extensions are loaded in a different way with the `extensionDevelopmentPath` is specified. I guess it is needed for letting c8 obtain code coverage-info.